cmd/juju/model/dump_test.go

```diff
@@ -32,14 +32,15 @@ func (f *fakeDumpClient) Close() error {
 	return f.NextErr()
 }
 
-func (f *fakeDumpClient) DumpModel(model names.ModelTag) (map[string]interface{}, error) {
+func (f *fakeDumpClient) DumpModel(model names.ModelTag, simplified bool) (map[string]interface{}, error) {
```

Comment:
We should record the value of simplified and in TestDumpSimple verify that it was indeed as expected